test/unit/v2/connectors/test_fsspec_paths.py

```diff
@@ -0,0 +1,93 @@
+from unittest.mock import MagicMock, patch
+from urllib.parse import unquote
+
+import pytest
+
+from unstructured_ingest.v2.processes.connectors.fsspec.fsspec import (
+    FsspecAccessConfig,
+    FsspecConnectionConfig,
+    FsspecIndexer,
+    FsspecIndexerConfig,
+    Secret,
+)
+
+
+@pytest.mark.parametrize(
+    ("remote_url", "expected_path"),
+    [
+        ("dropbox://da ta", "da ta"),
+        ("dropbox://da%20ta", "da ta"),
+    ],
+)
```

Comment:
What if my dropbox folder is named "a%20b" (literal name, not encoded)? Will it return `dropbox://a%20b` or `dropbox://a b`?